x/ccv/provider/keeper/consumer_lifecycle.go

```diff
@@ -330,8 +330,14 @@ func (k Keeper) MakeConsumerGenesis(
 			"consumerId", consumerId,
 			"minPower", minPower,
 		)
-		k.OptInTopNValidators(ctx, consumerId, activeValidators, minPower)
+
+		// set the minimal power of validators in the top N in the store
 		k.SetMinimumPowerInTopN(ctx, consumerId, minPower)
+
```

Comment:
Does order of these operations matter?
Previously, validators would be opted in before power. Here, the power is set before getting the opt in?

Reply:
No, it does not. I wanted to be consistent with the same operations performed in QueueVSCPackets